Chatify/chat/api.py

```diff
@@ -103,16 +103,6 @@ def get(self, request):
         return Response(status=status.HTTP_200_OK)
 
 
-class OnlineUsersAPI(APIView):
-    def get(self, request):
-        data = (
-            User.objects.filter(is_active=True)
-            .order_by("-id")
```

Comment:
Why have you removed this?

Reply:
in line no. 79 there is already a class.Here 2nd time class is added that's why i removed.